lib/ace/mode/latex_parser.js

```diff
@@ -1,30 +1,59 @@
 define(function(require, exports, module) {
 "use strict";
 
-var WORD_PATTERN = "[a-zA-Zа-яА-ЯёЁ]+";
-var WORD_REGEX = new RegExp("^" + WORD_PATTERN);
+var WORD_REGEX = /^[a-zа-яё]+/i,
+    // % * <foo@bar.com> 2015-01-12T00:17:27.829Z:
+    DISCUSSION_COMMENT_REGEXP = /^\s*%\s([\*\^])\s<(.+)>(\s(.+))?:$/,
```

Comment:
> Maybe include \s into .+ in the last group and make it just .*? 

@gkalabin I want to parse datetime without leading spaces. Seems like `(\s+(.+))?` is the easiest way. It allows no datetime (for handwritten discussions), and there always be at least one space between author and datetime.